pom.xml

```
@@ -68,7 +68,6 @@
     </repositories>
     <name>clas12mon</name>
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <release>21</release>
```

This doesn't do anything; in fact, it produces `.class` files with major version 52 (Java 8). The correct property is:

```suggestion
        <maven.compiler.release>21</maven.compiler.release>
```